ci/dash/lint.sh

```
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+# Wrapper script to run lint - created to fix CI path issue
+cd "$(dirname "$0")/../.."
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)